en/_posts/2024-06-05-unified-design-system.md

```diff
@@ -0,0 +1,37 @@
+---
+altLangPage: /2024/06/05/systeme-design-unifie.html
+css:
+-  ../../../css/blog.css
+- https://use.fontawesome.com/releases/v5.15.4/css/all.css
+date: 2024-06-05
+dateModified: 2024-06-05
+description: "Learn about how we're creating a unified source for code, design and documentation to help teams build websites and applications."
+title: "Unified design system and update of DTO’s merger with CDS"
+---
+
+<h2>A path toward a unified design system for the Government of Canada</h2>
+<p>One way to contribute to consistency in service delivery is through the use of a cohesive design system. In the Government of Canada, we’re working to develop an updated design system to meet evolving needs of service delivery teams and the public, in a digital age.</p>
+
+<p><b>Read more:</b></p>
+<div class="col-md-12 mrgn-bttm-xl">
+<div class="col-md-6">
+        <img class="img-responsive thumbnail" src="/images/DTO_CDS_DesignSystem_Blog_Post_EN.jpg" alt=""/>
+        <h3><a class="stretched-link" href="https://digital.canada.ca/2024/05/27/a-path-toward-a-unified-design-system-for-the-government-of-canada/">A path toward a unified design system for the Government of Canada</a></h3>
+</div>
+</div>
+
+
+<div class="clearfix">
```

Comment:
```suggestion
```

Reply:
Clearfix are no longer necessary since the grid system doesn't rely on floats as much.